### PR TITLE
Legacy deprecation banner update

### PR DESCRIPTION
**Pull request by austensen**
As part of our WOW legacy deprecation plan we are updating the copy on the banner, and adding a note to the about page. 
We also remove the "new" pill for the search by landlord name feature on the homepage. 

[sc-11583]

### CODE REVIEW

File: client/src/containers/App.tsx [hunk 1]
```diff
@@ -298,16 +298,16 @@ const WowzaBanner = withI18n()((props: withI18nProps) => {
...
               Switch to new version.
             </ToggleLinkBetweenPortfolioMethods>
           </Trans>
         ) : (
           <Trans>
-            Starting March 2023{" "}
+            In April 2023, this link to{" "}
             <ToggleLinkBetweenPortfolioMethods>the old version</ToggleLinkBetweenPortfolioMethods>{" "}
-            of Who Owns What will no longer be available.
+            of Who Owns What will will move to the About Page.
```
Comment (kiwansim) [inline client/src/containers/App.tsx:310]: two instances of 'will'

Comment (austensen) [reply client/src/containers/App.tsx]: ugh sorry - thanks for catching

File: client/src/locales/es/messages.po [hunk 2]
```diff
@@ -51,7 +51,7 @@ msgstr "(esto puede tardar un rato)"
 msgid "({browserType, select, mobile {tap} other {click}} to view details)"
 msgstr "({browserType, select, mobile {pulsa} other {haz clic}} para ver detalles)"
 
-#: src/containers/HomePage.tsx:123
```
Comment (kiwansim) [inline client/src/locales/es/messages.po:54]: are these line number references automatically changed when `.tsx` files are modified?

**Review by kiwansim — APPROVED** [review]
looks good to me! 🥇

File: client/src/locales/es/messages.po (hunk 2 above)
Comment (austensen) [reply client/src/locales/es/messages.po:54]: yeah, you just run `yarn lingui extract` and these files are regenerated. Some of the details of exactly how it keeps track of everything and what the `#~` type things signify I'm not actually sure about.

**Pull request merged**